File: specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2023-11-01-preview/examples-js/AscResourceUsages_Get.js
const { StorageCacheManagementClient } = require("@azure/arm-storagecache");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Gets the quantity used and quota limit for resources
 *
 * @summary Gets the quantity used and quota limit for resources
 * x-ms-original-file: specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2023-11-01-preview/examples/AscResourceUsages_Get.json
 */
async function ascUsagesList() {
  const subscriptionId =
    process.env["STORAGECACHE_SUBSCRIPTION_ID"] || "00000000-0000-0000-0000-000000000000";
  const location = "eastus";
  const credential = new DefaultAzureCredential();
  const client = new StorageCacheManagementClient(credential, subscriptionId);
  const resArray = new Array();
  for await (let item of client.ascUsages.list(location)) {
    resArray.push(item);
  }
  console.log(resArray);
}

File: specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2024-03-01/examples-java/AscResourceUsages_Get.java

/**
 * Samples for AscUsages List.
 */
public final class Main {
    /*
     * x-ms-original-file:
     * specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2024-03-01/examples/
     * AscResourceUsages_Get.json
     */
    /**
     * Sample code: AscUsages_List.
     * 
     * @param manager Entry point to StorageCacheManager.
     */
    public static void ascUsagesList(com.azure.resourcemanager.storagecache.StorageCacheManager manager) {
        manager.ascUsages().list("eastus", com.azure.core.util.Context.NONE);
    }
}
